samples/dml.js

```diff
@@ -529,6 +529,61 @@ async function deleteUsingPartitionedDml(instanceId, databaseId, projectId) {
   // [END spanner_dml_partitioned_delete]
 }
 
+function updateUsingBatchDml(instanceId, databaseId, projectId) {
+  // [START spanner_dml_batch_update]
+  // Imports the Google Cloud client library
+  const {Spanner} = require('@google-cloud/spanner');
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Creates a client
+  const spanner = new Spanner({
```

Comment:
do we need to explicitly define the projectId here?

Reply:
This matches working client instantiation already existing in sample. I'd prefer to address this in a separate PR if possible.